R/stat-function.r

```diff
@@ -16,39 +19,44 @@
 #'   \item{x}{x's along a grid}
 #'   \item{y}{value of function evaluated at corresponding x}
 #' }
+#' @seealso \code{\link[rlang:as_function]{as_function()}}
```

Comment:
Should this be `rlang::as_function`? (two colons instead of one)

Reply:
I thought in the link you put a single colon; works in the docs as I built them, at least.

Reply:
A single colon is correct.

c.f. https://cran.r-project.org/doc/manuals/R-exts.html#Cross_002dreferences

Reply:
Sorry, I was not accurate above. A single colon is correct as a raw `Rd` code, but we should use Markdown-style as we do so in other places: `[rlang::as_function]`.

Reply:
Looks right with parentheses. (I'm seeing more like `[rlang::as_function()]`.) Got it.

Reply:
Yes, `[rlang::as_function()]` is correct, sorry. More details are here: https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html#links